pkg/util/export/etl/tae.go

```diff
@@ -246,17 +247,24 @@ func getOneRowData(ctx context.Context, bat *batch.Batch, Line []any, rowIdx int
 		case types.T_json:
 			switch t := field.(type) {
 			case string:
-				byteJson, err := types.ParseStringToByteJson(field.(string))
-				if err != nil {
-					return moerr.NewInternalError(ctx, "the input value is not json type for column %d: %v", colIdx, field)
-				}
-				jsonBytes, err := types.EncodeJson(byteJson)
-				if err != nil {
-					return moerr.NewInternalError(ctx, "the input value is not json type for column %d: %v", colIdx, field)
-				}
-				err = vector.SetBytesAt(vec, rowIdx, jsonBytes, mp)
-				if err != nil {
-					return err
+				if ctx.Value(types.T_json) != true {
+					byteJson, err := types.ParseStringToByteJson(field.(string))
```

Comment:
this segment should be deleted, base on L121 always set context.Value(Type.Tjson) = true